README.md

```diff
@@ -91,6 +92,8 @@ Check msr 0x503, return value must be 0:
 sudo rdmsr 0x503s
 ```
 
+Note that if you want to run SEV-SNP remote attestation, please refer to [link](https://github.com/AMDESE/AMDSEV/tree/sev-snp-devel) to set up the host and guest Linux kernel, qemu and ovmf bios used for launching SEV-SNP guest.
+
```

Comment:
suggest adding some notice about bios version and cmdline setttings etc.

Reply:
There are many components to start the snp guest(bios, psp version, host kernel, guest kernel, ovmf, qemu,etc), and they have not been upstream. This information are described in detail in the [link](https://github.com/AMDESE/AMDSEV/tree/sev-snp-devel), and are not mentioned in the readme

Reply:
ok, thanks